.gitignore

```diff
@@ -54,6 +54,7 @@ docs/_build/
 
 # Jupyter Notebook
 .ipynb_checkpoints
+test_yellowbrick.ipynb
```

Comment:
I think it's an excellent idea to modify your local .gitignore to include your test notebook and scripts so they don't accidentally get committed with the rest of your PR; but if possible, we'd like to keep the global .gitignore as lightweight as possible (we have lots of contributors with lots of different preferences about what these files are named, e.g. I use `pad.py` or `t.py`). Would you mind reverting the `.gitignore` file back to the original for this PR?

As an alternative, note that you can put notebooks and scripts (and commit them) into `examples/rohit-ganapathy` if that would make testing and development easier.